NetUtils.py

```diff
@@ -106,6 +106,17 @@ def _scan_for_TypedTuples(obj: typing.Any) -> typing.Any:
     return obj
 
 
+def convert_to_base_types(obj: typing.Any) -> typing.Any:
```

Comment:
non-blocking: do we want docstrings on functions in general? The typing doesn't really help for this function and it might be good to explain what the intention for it is